### ISSUE TITLE
#imageLiteral handling bug

### ISSUE THREAD

**Issue by xuaninbox** [nicklockwood/SwiftFormat#283]
When using #imageLiteral, there's a bug where

#imageLiteral(resourceName: "abc.pdf").pngData()

will be formatted to

#imageLiteral(resourceName: "abc.pdf").pngData

**Comment by nicklockwood** [comment]
I'm not able to reproduce this with just that line. Can you provide a larger code sample that demonstrates the issue?

**Comment by xuaninbox** [comment]
```
import UIKit

class Test: NSObject {

    func test() {
        if let data = #imageLiteral(resourceName: "common_logo.pdf").pngData() {
            print(data)
        }
    }
}
```

it seems only happen when there's nothing after (), this does not trigger
```
guard let data = #imageLiteral(resourceName: "common_logo.pdf").pngData() else {
    return
}
```

**Comment by nicklockwood** [comment]
@xuaninbox fixed in 0.34.0
